dashboard/app/models/concerns/pd/facilitator_specific_form.rb

```diff
@@ -89,4 +89,14 @@ def form_data_hash=(hash)
 
     super(hash)
   end
+
+  def to_summary_for_facilitator(facilitator_name)
```

Comment:
nit: I'm not sure "to_summary" is particularly clear here. Maybe something like "filter_summary_for_facilitator"?